modules/ROOT/pages/managing-permissions.adoc

```diff
@@ -8,49 +8,49 @@ This section explains how permissions work across different products and APIs ma
 
 == Assumptions
 
-This page assumes that you have an Organization Administrator role in your organization, that you have been assigned as the administrator of one of the business groups of your Organization, or that you have API Version Owner permissions and that you want to manage user permissions for an API version, a business group or the entire organization.
+This page assumes that you have an Organization Administrator permission in your organization, that you have been assigned as the administrator of one of the business groups of your Organization, or that you have API Version Owner permissions and that you want to manage user permissions for an API version, a business group or the entire organization.
 
-== How Permissions Work in the Anypoint Platform
+== How Permissions Work in Anypoint Platform
 
-In the Anypoint Platform, users belong to an organization and have a set of *roles* and *permissions*.
+In the Anypoint Platform, users belong to an organization and have a set of permissions. These permissions are assigned using xref:teams.adoc[teams], xref:roles.adoc[roles] (deprecated), or by xref:users.adoc#grant-user-permissions[granting permissions to users individually].
```

Comment:
Should this be plural, because users can (do) belong to more than one org and have more than one permission?
```suggestion
In the Anypoint Platform, users belong to organizations and have sets of permissions. These permissions are assigned using xref:teams.adoc[teams] or xref:roles.adoc[roles] (deprecated), or by xref:users.adoc#grant-user-permissions[granting permissions to users individually].
```
This is probably a good place to explain what the deprecation of Roles functionality means....